transforms/React-DOM-to-react-dom-factories.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+'use strict';
+
+module.exports = function(file, api, options) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+
+  let hasModifications;
+
+  const DOMModuleName = 'DOM';
+
+  const isDOMSpecifier = specifier => (
+    specifier.imported &&
+    specifier.imported.name === DOMModuleName
+  );
+  const removeDeconstructedDOMStatement = (j, root) => {
```

Comment:
maybe a more accurate name would be 'replaceDeconstructedDOMStatement'.

In each case it replaces it with 'createElement'.